server/controllers/bookmark.contoller.js

```diff
@@ -0,0 +1,33 @@
+import db from '../models';
+import Response from '../utils/response.util';
+
+const { Bookmark, Article } = db;
+/**
+ * Bookmark Controller
+ * @package bookmark
+ */
+class BookmarkController {
+  /**
+   * @description - Controller for creating a bookmark endpoint
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} response
+  */
+  static async createOrRemoveBookmark(req, res) {
+    const {
+      articleId
+    } = req.params;
+    const { id } = req.user;
+    const isArticleExists = await Article.findOne({ where: { id: articleId } });
```

Comment:
@samorahno There is a util function to check if the article does not exist, why not use that as a middleware?

Reply:
@davidokonji you are right, but the util function existing does not fit into this. it uses req.body

Reply:
Yes, I observed same @davidokonji. The util function gets id from the body but he is getting from the params